Gruntfile.js

```diff
@@ -19,7 +19,7 @@ module.exports = function(grunt) {
         dest: 'lib/tslint.js'
       },
       test: {
-        src: ['<%= typescriptBin %>', 'build/tslint-tests.js'],
+        src: ['lib/tslint.js', 'build/tslint-tests.js'],
```

Comment:
Tests were getting compiled with the source included because of a reference to `../src/tslint.ts` in `test/helper.ts`. Since we removed reference paths, that part of the built `build/tslint-tests.js` file got lost. I didn't want to have to re-compile `src` every time `grunt test` is run, so I changed this line. Open to other ideas